src/cogs/redditAPI.py

```diff
@@ -30,7 +30,15 @@ async def on_ready(self):
     @commands.command()
     async def memes(self, ctx, arg : int, x = 1):
         
-        reddit = asyncpraw.Reddit(client_id = "Vng40QEkdlR_VuVphsbxxA", client_secret = "4evSCjg47N9CTtQqqyQUuA8X2I4qXQ", username = "ProfessionalFloor135",  password =  "Qwerty@18606", user_agent = 'EduMemebot')
+        with open('./cogs/credentials.txt','r') as file_8:
+            s = file_8.readlines()
+
+        reddit = asyncpraw.Reddit(client_id = str(s[0]).strip("\n"), 
+        client_secret = str(s[1]).strip("\n"), 
+        username = str(s[2]).strip("\n"),
+        password =  str(s[3]).strip("\n"), 
+        user_agent = str(s[4]).strip("\n"))
```

Comment:
Using a simple json would be better for structured data. Here is a implementation off the top of my head. Not sure if it will work.

```python
with open('./cogs/credentials.txt','r') as auth_file:
  auth_dict_json = auth_file.read()

auth_dict = json.loads(auth_dict_json)
reddit = asyncpraw.Reddit(**auth_dict)
```